content/docs/csidriver/troubleshooting/powerflex.md

```diff
@@ -24,6 +24,9 @@ description: Troubleshooting PowerFlex Driver
 | CSI-PowerFlex volumes cannot mount; are being recognized as multipath devices | CSI-PowerFlex does not support multipath; to fix: <br/> 1. Remove any multipath mapping involving a powerflex volume with `multipath -f <powerflex volume>` <br/> 2. Blacklist CSI-PowerFlex volumes in multipath config file |
  | When attempting a driver upgrade, you see: ```spec.fsGroupPolicy: Invalid value: "xxx": field is immutable``` |  You cannot upgrade between drivers with different fsGroupPolicies. See [upgrade documentation](../../upgradation/drivers/powerflex) for more details | 
  | When accessing ROX mode PVC in OpenShift where the worker nodes are non-root user, you see: ```Permission denied``` while accesing the PVC mount location from the pod. | Set the ```securityContext``` for ROX mode PVC pod as below, as it defines privileges for the pods or containers.<br/><br/>securityContext:<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;runAsUser: 0<br/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;runAsGroup: 0 |
+| After installing version v2.6.0 of the driver on Ubuntu or RHEL 8.3 machines using the default sdc:3.6.0.6 `powerflexSdc` image, the vxflexos-node pods are in an `Init:CrashLoopBackOff` state. | Change the `images.powerflexSdc` value to `dellemc/sdc:3.6` in the [values](https://github.com/dell/csi-powerflex/blob/72b27acee7553006cc09df97f85405f58478d2e4/helm/csi-vxflexos/values.yaml#L13) and re-install. |
```

Comment:
One caveat: Ubuntu does not support using sdc:3.6.0.6. Init containers are only supported on RHEL 8.4+ and RHCOS environments. That being said, PowerFlex SDC 3.6 was tolerant of situations where SDC was installed already and would gracefully exit. That is not the case with SDC 3.6 and it is indeed tolerant. If we change to using SDC 3.6 we are only telling to use an SDC that is tolerant to an already installed SDC. They could very easily not provide any SDC value and things should just work. I am leaning on a solution that tells the users to deploy SDC manually and not use any SDC init container image in values file instead of asking them to switch the SDC image, if you need to discuss more please reach out to me.

Reply:
Updated the workaround to specify an empty `powerflexSdc` value.